data/json/items/comestibles/carnivore.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "abstract": "flesh",
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "name": "meat/fish",
+    "material": "flesh",
+    "symbol": "%",
+    "color": "red",
+    "parasites": 32,
+    "vitamins": [
+      [ "calcium", 1 ],
+      [ "iron", 20 ],
+      [ "vitB", 10 ]
+    ]
+  },
+  {
+    "id": "fish",
+    "copy-from": "flesh",
+    "type": "COMESTIBLE",
+    "name": "fillet of fish",
+    "name_plural": "fillets of fish",
+    "description": "Freshly caught fish.  Makes a passable meal raw.",
+    "weight": 200,
+    "volume": 1,
+    "price": 500,
+    "spoils_in": 12,
+    "calories": 200
+  },
+  {
+    "id": "fish_cooked",
+    "type": "COMESTIBLE",
+    "name": "cooked fish",
+    "name_plural": "cooked fish",
+    "description": "Freshly cooked fish.  Very nutritious.",
+    "proportional": { "price": 1.5 },
+    "spoils_in": 24,
+    "parasites": 0,
+    "flags": [ "EATEN_HOT" ]
+  },
+  {
+    "id": "hstomach",
+    "copy-from": "stomach",
+    "type":"COMESTIBLE",
+    "name": "human stomach",
+    "description": "The stomach of a human.  It is surprisingly durable.",
+    "material": "hflesh"
+  },
+  {
+    "id": "hstomach_large",
+    "copy-from": "hstomach",
+    "type":"COMESTIBLE",
+    "name": "large human stomach",
+    "description": "The stomach of a large humanoid creature.  It is surprisingly durable.",
+    "proportional": {
+      "weight": 2.0,
+      "volume": 2.0,
+      "price": 1.5,
+      "calories": 2.0
+    }
+  },
+  {
+    "id": "human_flesh",
+    "copy-from": "meat",
+    "type": "COMESTIBLE",
+    "name": "human flesh",
+    "name_plural": "human fleshes",
+    "description": "Freshly butchered from a human body.",
+    "material": "hflesh"
```

Comment:
This will affect some already merged content also, I'll see if I can work it out